### ISSUE TITLE
sys-apps/man-1.6g-r3 - l10n.eclass could not be found by inherit()

### ISSUE THREAD

**Issue by ultratensai** [vaeth/mv-overlay#24]
```
coruscant ~ # emerge -uUDva @world

These are the packages that would be merged, in order:

Calculating dependencies \ * ERROR: sys-apps/man-1.6g-r3::mv failed (depend phase):
 *   l10n.eclass could not be found by inherit()
 *
 * Call stack:
 *            ebuild.sh, line 645:  Called source '/var/db/repos/mv/sys-apps/man/man-1.6g-r3.ebuild'
 *   man-1.6g-r3.ebuild, line   6:  Called inherit 'l10n' 'prefix' 'toolchain-funcs'
 *            ebuild.sh, line 295:  Called die
 \ * The specific snippet of code:
 *              [[ -z ${location} ]] && die "${1}.eclass could not be found by inherit()"
 *
 * If you need support, post the output of `emerge --info '=sys-apps/man-1.6g-r3::mv'`,
 * the complete build log and the output of `emerge -pqv '=sys-apps/man-1.6g-r3::mv'`.
 * Working directory: '/usr/lib/python3.9/site-packages'
 * S: '/var/tmp/portage/sys-apps/man-1.6g-r3/work/man-1.6g'
... done!

```

**Comment by ultratensai** [comment]
it seems l10n.eclass has been [deprecated](https://www.mail-archive.com/gentoo-dev@lists.gentoo.org/msg92587.html).

```
obiwan at coruscant in /var/db/repos/gentoo/eclass (stable)
$ ls -alF /var/db/repos/gentoo/eclass/ | grep 10n

obiwan at coruscant in /var/db/repos/gentoo/eclass (stable)
$
```

**Comment by vaeth** [comment]
Fixed in commit 899d752c123aa9df8ba4aab50c7627cf71b7d415
